packages/Webkul/Admin/src/Resources/lang/vi/app.php

```diff
@@ -0,0 +1,4014 @@
+                'greeting'    => 'Chúng tôi gửi lời chào nồng nhiệt đến khách hàng mới, :customer_name, người vừa đăng ký với chúng tôi!',
+                'subject'     => 'Đăng ký Khách hàng Mới',
+            ],
+        ],
+
+
+        'orders' => [
+            'created' => [
+                'greeting' => 'Bạn có một đơn hàng mới :order_id được đặt vào :created_at',
+                'subject'  => 'Xác nhận Đơn hàng Mới',
+                'summary'  => 'Tóm tắt Đơn hàng',
+                'title'    => 'Xác nhận Đơn hàng!',
+            ],
+
+            'invoiced' => [
+                'greeting' => 'Hóa đơn của bạn #:invoice_id cho đơn hàng :order_id được tạo vào :created_at',
+                'subject'  => 'Xác nhận Hóa đơn Mới',
+                'summary'  => 'Tóm tắt Hóa đơn',
+                'title'    => 'Xác nhận Hóa đơn!',
+            ],
+
+            'shipped' => [
+                'greeting' => 'Bạn đã gửi hàng cho đơn hàng :order_id được đặt vào :created_at',
+                'subject'  => 'Xác nhận Giao hàng Mới',
+                'summary'  => 'Tóm tắt Giao hàng',
+                'title'    => 'Đơn hàng Đã được Gửi!',
+            ],
+
+            'inventory-source' => [
+                'greeting' => 'Bạn đã gửi hàng cho đơn hàng :order_id được đặt vào :created_at',
+                'subject'  => 'Xác nhận Giao hàng Mới',
+                'summary'  => 'Tóm tắt Giao hàng',
+                'title'    => 'Đơn hàng Đã được Gửi!',
+            ],
+
+            'refunded' => [
+                'greeting' => 'Bạn đã hoàn tiền cho đơn hàng :order_id được đặt vào :created_at',
+                'subject'  => 'Xác nhận Hoàn tiền Mới',
+                'summary'  => 'Tóm tắt Hoàn tiền',
+                'title'    => 'Đơn hàng Đã được Hoàn Tiền!',
+            ],
+
+            'canceled' => [
+                'greeting' => 'Bạn đã hủy đơn hàng :order_id được đặt vào :created_at',
+                'subject'  => 'Đơn hàng Mới đã bị Hủy',
+                'summary'  => 'Tóm tắt Đơn hàng',
+                'title'    => 'Đơn hàng Đã bị Hủy!',
+            ],
+
+            'billing-address'   => 'Địa chỉ Thanh toán',
+            'contact'           => 'Liên hệ',
+            'discount'          => 'Giảm giá',
+            'grand-total'       => 'Tổng cộng',
+            'name'              => 'Tên',
+            'payment'           => 'Thanh toán',
+            'price'             => 'Giá',
+            'qty'               => 'Số lượng',
+            'shipping'          => 'Vận chuyển',
+            'shipping-address'  => 'Địa chỉ Giao hàng',
+            'shipping-handling' => 'Xử lý Giao hàng',
+            'sku'               => 'SKU',
+            'subtotal'          => 'Tổng phụ',
+            'tax'               => 'Thuế',
+        ],
+
```

Comment:
Please remove extra space and match all the arrays some extra space and keys are added.
take the reference from 'en' (english) language. Number of lines should be same.